README.md

```diff
@@ -110,7 +110,12 @@ JSONSchemer.schema(
   # 'net/http'/proc/lambda/respond_to?(:call)
   # 'net/http': proc { |uri| JSON.parse(Net::HTTP.get(uri)) }
   # default: proc { |uri| raise UnknownRef, uri.to_s }
-  ref_resolver: 'net/http'
+  ref_resolver: 'net/http',
+  
+  # uses different method to match regexes
+  # Proc/lambda/respond_to?(:call)
+  # default: Regexp.method(:new)
+  regexp_resolver: ->(pattern) { RE2::Regexp.new(pattern) }
```

Comment:
Let's use `proc` here instead of a lambda to be consistent with the other examples.